3/ollie-charles-captions.srt

```diff
@@ -2204,7 +2206,7 @@ I should be able to know
 
 509
 00:32:50,413 --> 00:32:52,600
-(inaudible) problems.
+[inaudible] of one of the example problems.
```

Comment:
This sounds to me like:

    If I load the dictionary,
    I should be able to now
    try one of the example problems.